python/lsst/cp/verify/verifyBias.py

```diff
@@ -89,9 +97,10 @@ def verify(self, exposure, statisticsDict):
                 verify['PROCESSING'] = False
 
             verify['SUCCESS'] = bool(np.all(list(verify.values())))
-
             if verify['SUCCESS'] is False:
                 success = False
+            # This isn't something to fail on yet.
```

Comment:
So should this actually be a `TODO` with an associated ticket, or should we provide more information? I'm trying to think of our future selves when we come back to this part of the code and wonder about this (i.e., when/if should we fail on this?)

Reply:
I've updated the comment.  I think we would only want to fail on this if we implement versioned cameras.